content/runbooks/alertmanager/AlertmanagerConfigInconsistent.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Alertmanager ConfigInconsistent
+title: Alertmanager Config Inconsistent
```

Comment:
WDYT about keeping it consistent with alert name in prometheus and removing spaces instead of adding them?

Reply:
Yeah was wondering about it:
- with spaces - page title with spaces will be on the left side of the web page and it is easier to read
- without spaces - some alert names are really long and this may look ugly there.

On the other hand first header in the page is left as is.

Also I will check if searching for alerts from the search bar works with names with spaces and without it.
I would rather keep it with spaces and hid somwhere the name without spaces :)

Reply:
Removing entries without spaces makes them impossible to find via such short alert name, which is a bit problematic.

Will have to look into the Hugo options or something.